cips/cip-start-times.md

```diff
@@ -0,0 +1,60 @@
+
+## Motivation
+
+Currently, the time it takes to commit to a block is propotional to the size of that block (largely owing to the block propagation mechansim). Celestia currently has an 11 second timeout commit which means when blocks are near empty we see block times of around 11.5 seconds, but when the network is put under load with 8MB blocks, this rises to 18 seconds. This CIP proposes adjustments such that the average interval between block times is consistent (based on a `TargetInterval` parameter). This however does not expect to impact the standard deviation of block times.
+
+## Specification
+
+The following modifications are proposed:
+
+- Validators select a vote time that is their `cs.StartTime` as opposed to their current time when they vote for a block.
+  - In the case, that `cs.StartTime` is after the locked or proposal block time, the validator will pick a time that is 1 millisecond later to satisfy the property of increasing block times.
+  - In the case of the first block, the start time will be the current time.
+- Upon receiving 2f + 1 PRECOMMIT votes, each validator calculates the **local** stake-weighted median start time from those votes. It sets the new start time of the next height to be the aforementioned calculated time plus the `TargetInterval`.
+  - If the nodes local time is after the new start time, then the node will start immediately. This will be the case for heights that require multiple rounds to come to consensus.
+- A proposer MUST include a block time in the header that is equal to the stake-weighted median start time of the precommit votes that are included in the LastCommit of the proposed block + the `TargetInterval`.
+  - All validators MUST verify that the block time has been calculated as per the above rules. If not, the block will not be accepted.
+- `TargetInterval` is part of `BlockParams` within `ConsensusParams`. A `TargetInterval` of zero signifies that this mechanism is disabled. The `TargetInterval` CAN be set in a hard fork upgrade when all nodes are running the latest version, activating the new timing mechnanism.
+- The `TargetInterval` once activated will be set to 12 seconds. `TargetInterval` can not be modified via governance and SHOULD only change in major version upgrades.
+
+## Parameters
+
+| Parameter     | Proposed value | Description                                                                                                                | Changeable via Governance |
+|---------------|---------|------------------------------------------------------------------------------------------------------------------------|---------------------------|
+| BlockParams.TargetIntervalMS | 12 seconds  | The target interval between start times in milliseconds                                                            | false                     |
```

Comment:
nit: the param name and description use milliseconds but the proposed value is in seconds. Proposal to consolidate on one unit.

If it's milliseconds, you could put 12 seconds in parenthesis like: `12000 milliseconds (12 seconds)`